src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class Cars {
+    private static final int MINIMUM_NUMBER_OF_CARS = 2;
+    private static final int MAXIMUM_NUMBER_OF_CARS = 50;
+
+    private final List<Car> racingCars;
+
+    public Cars(final String inputName) {
+        final List<String> names = List.of(inputName.split(","));
```

Comment:
입력받은 이름을 `,` 구분자로 구분한다는 것이 도메인이 알고있어야할 내용일지 고민해봐야겠어요 🤔 

1. 만약 View 에서 입력하는 형태에 구분자가 변경됐다고 해서 도메인에서 에러가 발생할 가능성을 놔두는것이 맞을까요?
2. 이름들 이라는 문자열을 구분자로 나누는 과정이 도메인 로직에 존재한다는 것을 쉽게 예측 가능할까요?

Reply:
Controller에서 변환하는 방식으로 수정했습니다 !!

웹 컨트롤러에서 Request Body로 받아올 때 리스트로 받아오려고 했습니다.
하지만 Json으로 변환할 때 곧바로 String을 split할 수 없었습니다.
`Json 변환 방식을 직접 추가하려고 했으나 default convert 방식에 영향이 있을 수 있습니다.`
만약 Json 변환 방식을 직접 추가할 경우 계속해서 저의 웹 자동차 경주에서는 그런 방법이 가능하다는 문서도 필요하게 되어 번거로움이 있습니다. 

그래서 결국 Controller에서 변환하는 방식으로 수정했습니다.
Cars라는 도메인 객체는 플레이어 이름 값을 리스트 형식으로 받는 것은 고정시켰습니다.
덕분에 View에서 어떤 입력의 형태가 오더라도 Controller에서 직접 List 타입으로 변환시켜서 넘겨주면 됩니다.
`이럴 경우 도메인에서 에러가 발생하지도 않고 View 변경의 영향이 도메인까지 들어오지 않는다는 장점이 있었습니다.`

이러한 과정을 바탕으로 다음과 같이 수정했습니다.

```suggestion
    public Cars(final List<String> nameValues) {
        // ...
    }
```

Reply:
잘 반영해주셨네요 👍 
말씀대로 Json 변환 방식을 변경하는 것은 다른 개발자가 쉽게 알아차릴 수 없는 형태가 될 것같다는 걱정이 드네요!

보통 requestBody의 값은 Client 와 어떤 형태로 값을 받을지 협의할테고 
값을 어떻게 처리해서 비즈니스 로직을 실행할지만 다른 개발자가 명확하게 이해할 수 있도록 만들어주면 된다고 생각해요!